cam_base_topic: camera/image_raw
output_frame: ''
publish_tf: true
marker_size: 0.100
image_queue_size: 1

# Dynamically reconfigurable Detector parameters
# https://docs.opencv.org/4.2.0/d5/dae/tutorial_aruco_detection.html
adaptiveThreshWinSizeMin: 3
adaptiveThreshWinSizeMax: 23
adaptiveThreshWinSizeStep: 10
adaptiveThreshConstant: 7
minMarkerPerimeterRate: 0.03
maxMarkerPerimeterRate: 4.0
polygonalApproxAccuracyRate: 0.03
minCornerDistanceRate: 0.05
minDistanceToBorder: 3
minMarkerDistanceRate: 0.05
markerBorderBits: 1
perspectiveRemovePixelPerCell: 4
perspectiveRemoveIgnoredMarginPerCell: 0.13
maxErroneousBitsInBorderRate: 0.35
minOtsuStdDev: 5.0
errorCorrectionRate: 0.6
cornerRefinementMethod: 0 # 0 - None, 1 - Subpix, 2 - Contour, 3 - Apriltag
cornerRefinementWinSize: 5
cornerRefinementMaxIterations: 30
cornerRefinementMinAccuracy: 0.1
